system_tests/speech.py

```diff
@@ -22,10 +22,34 @@
 
 from system_test_utils import unique_resource_id
 from retry import RetryErrors
+from retry import RetryResult
 
 AUDIO_FILE = os.path.join(os.path.dirname(__file__), 'data', 'hello.wav')
 
 
+def _operation_complete(result):
+    """Return operation result."""
+    return result
+
+
+def _wait_until_complete(operation, max_attempts=5):
+    """Wait until an operation has completed.
+
+    :type operation: :class:`google.cloud.operation.Operation`
+    :param operation: Operation that has not completed.
+
+    :type max_attempts: int
+    :param max_attempts: (Optional) The maximum number of times to check if
+                         the operation has completed. Defaults to 5.
+
+    :rtype: bool
+    :returns: Boolean indicating if the operation is complete.
+    """
+
```

Comment:
(FWIW, feel free to leave it here, but I doubt you meant to when you moved `_operation_complete` out)